src/test/java/eu/kandru/luna/controller/AuthControllerTest.java

```diff
@@ -36,6 +42,9 @@
 
```

Comment:
Please use this test class to assert that a password is actually sent to the correct client